app/services/vacancy_filter.rb

```diff
@@ -0,0 +1,31 @@
+class VacancyFilter
```

Comment:
Could we scope this to HiringStaff, given that we will need jobseeker filters too, which are different. Or we could do that for jobseeker filters?